### PR TITLE
LPS-115816 get translated content and add or replace to article content

### PR DESCRIPTION
**Pull request by AliciaGarciaGarcia**

### CODE REVIEW

**Comment by AliciaGarciaGarcia** [comment]
ci:test:relevant

**Comment by AliciaGarciaGarcia** [comment]
ci:test:sf

**Comment by AliciaGarciaGarcia** [comment]
ci:test:relevant

**Comment by AliciaGarciaGarcia** [comment]
ci:test:sf

**Comment by 4lejandrito** [comment]
Reviewing

File: modules/apps/translation/translation-test/src/testIntegration/java/com/liferay/translation/info/item/updater/test/InfoFormValuesUpdaterTest.java [hunk 1]
```diff
@@ -157,12 +230,131 @@ public void testUpdateArticleFromInfoFormValuesOnlyTitle()
...
+
+					return element.getStringValue();
+				}
+			}
+		}
+
+		return StringPool.BLANK;
+	}
+
+	private static final Locale _LOCALE_ES = LocaleUtil.SPAIN;
```
Comment (4lejandrito) [inline modules/apps/translation/translation-test/src/testIntegration/java/com/liferay/translation/info/item/updater/test/InfoFormValuesUpdaterTest.java:354]: Hi @AliciaGarciaGarcia,

Why do we need to extract these as constants? They are already constants and we don't do this in any other place.

I would use them explicitly without extracting them to constants. 

Also, if you really want to do it, it would be better to do it in a separate commit that is easily revertible if Brian doesn't like it.

File: modules/apps/translation/translation-test/src/testIntegration/java/com/liferay/translation/info/item/updater/test/InfoFormValuesUpdaterTest.java [hunk 2]
```diff
@@ -121,29 +119,96 @@ public void testUpdateArticleFromInfoFormValues() throws Exception {
...
+			descriptionMap, contentMap, LocaleUtil.getSiteDefault(), false,
+			true, _serviceContext);
+
+		InfoFormValues infoFormValues =
+			_xliffTranslationInfoFormValuesImporter.importXLIFF(
+				_group.getGroupId(),
+				new InfoItemClassPKReference(
+					JournalArticle.class.getName(), 122),
+				TranslationTestUtil.readFileToInputStream(
+					"test-journal-article_122_Jap.xlf"));
```
Comment (4lejandrito) [inline modules/apps/translation/translation-test/src/testIntegration/java/com/liferay/translation/info/item/updater/test/InfoFormValuesUpdaterTest.java:167]: Hi @AliciaGarciaGarcia, can you rename these files to use only underscores (`-`)?

Can we also use the language id (`ja-JP`) instead of `Jap` in the file name? That would be more precise and would set a better pattern if we want to add other languages in the future.

**Comment by 4lejandrito** [comment]
Hi @AliciaGarciaGarcia 

I think we should rename InfoFormValuesUpdater -> JournalArticleInfoFormValuesUpdater for now. Because otherwise we are using a generic name with specific parameters. I prefer to have the more explicit name published.

**Comment by AliciaGarciaGarcia** [comment]
close this and resent with changes on  https://github.com/liferay-lima/liferay-portal/pull/303
